tests/src/test/scala/whisk/core/containerpool/test/ContainerPoolTests.scala

```diff
@@ -68,7 +69,9 @@ class ContainerPoolTests extends TestKit(ActorSystem("ContainerPool"))
     val memoryLimit = 256.MB
 
     val invocationNamespace = EntityName("invocationSpace")
+    val invocationNamespace2 = EntityName("invocationSpace2")
     val action = ExecutableWhiskAction(EntityPath("actionSpace"), EntityName("actionName"), exec)
+    val action2 = ExecutableWhiskAction(EntityPath("actionSpace"), EntityName("actionName2"), exec)
```

Comment:
Same comment on naming. You can also write:

```
val differentAction = action.copy(name = EntityName("actionName2"))
```

Makes it a bit more apparent that you just change the name.